### PR TITLE
[dev] Review VerneMQ and its plugins' READMEs

### PR DESCRIPTION
**Pull request by eduardogmisiuk**
* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Is there any issue related to this PR?** (Link to an issue, e.g. #99999)
https://github.com/dojot/dojot/issues/1584

* **Other information**:

### CODE REVIEW

File: connector/mqtt/vernemq/broker/README.md [hunk 1]
```diff
@@ -1,55 +1,58 @@
 # **Dojot VerneMQ**
 
-The Dojot VerneMQ service is a extension of [VerneMQ](https://github.com/vernemq/vernemq) with some features for dojot case.
+The Dojot VerneMQ service is an extension of [VerneMQ](https://github.com/vernemq/vernemq) with some
+modifications for Dojot's integration.
 
+# **Configuration**
 
-## **Environment variables**
+There are two configurations for VerneMQ: the one provided by the VerneMQ itself and the Dojot's
```
Comment (rascaraficci) [inline connector/mqtt/vernemq/broker/README.md:8]: Shoudn't be better 'There are two configuration for the Dojot VerneMQ service: "?

File: connector/mqtt/vernemq/broker/README.md [hunk 2]
```diff
@@ -1,55 +1,58 @@
...
-CHECK_BROKER_CERT_REVOKED_TIME  | Checks if the public certificate of broker has revoked every time defined by this cron expression and renews them if necessary  | "0 */3 * * *" | cron schedule expressions
-CRL_UPDATE_TIME          | Retrieve the new CRL every time defined by this cron expression | "0 */2 * * *" | cron schedule expressions
-CHECKEND_EXPIRATION_SEC  | Check the certificates states and renews them if they expire within the next arg seconds | 43200  | seconds
+## **Dojot**
 
+Key                             | Purpose                                                                                     | Default Value           | Accepted values
+------------------------------- | ------------------------------------------------------------------------------------------- | ----------------------- |-----------------
+CA_NAME                         | CA Name from EJBCA                                                                          | IOTmidCA                | string
+CHECK_EXPIRATION_TIME           | Cron interval for certificate validation, renewing if necessary                             | 0 1 * * *               | cron expressions
```
Comment (rascaraficci) [inline connector/mqtt/vernemq/broker/README.md:16]: Wasn't this removed?

File: connector/mqtt/vernemq/broker/src/dojot_acl_plugin/README.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+```shell
+cd ../..
+docker build <image_name>:<tag>
+cd -
+```
+
+__NOTE THAT__ if you are not running a local VerneMQ instance, you should send the image to the
+machine that is running or use some kind of Docker Registry (e.g. DockerHub).
+
+After initializing this newly-made VerneMQ image, enter in its and enable the plugin:
```
Comment (rascaraficci) [inline connector/mqtt/vernemq/broker/src/dojot_acl_plugin/README.md:40]: enter in it

File: connector/mqtt/vernemq/broker/src/dojot_disconnect_plugin/README.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+```shell
+cd ../..
+docker build <image_name>:<tag>
+cd -
+```
+
+__NOTE THAT__ if you are not running a local VerneMQ instance, you should send the image to the
+machine that is running or use some kind of Docker Registry (e.g. DockerHub).
+
+After initializing this newly-made VerneMQ image, enter in its and enable the plugin:
```
Comment (rascaraficci) [inline connector/mqtt/vernemq/broker/src/dojot_disconnect_plugin/README.md:40]: enter in it

**Review by rascaraficci — COMMENTED** [review]
Good job! I left few comments.

File: connector/mqtt/vernemq/broker/README.md (hunk 2 above)
Comment (eduardogmisiuk) [reply connector/mqtt/vernemq/broker/README.md:16]: I think that the only thing we changed was the `update` parameter for the CRL retrieval. If we were to remove this here, we should have the sidecar service first, don't we?

Comment (rascaraficci) [reply connector/mqtt/vernemq/broker/README.md:16]: OK!

**Review by rascaraficci — APPROVED** [review]
It looks good to me!

**Review by thiagodpf — APPROVED** [review]

**Pull request merged**